### ISSUE TITLE
Support for Windows users

### ISSUE THREAD

**Issue by wint3ria** [eradiate/eradiate#373]
# Feature request

## Summary

Support for this OS should be trivial to implement but we lack a proper test infrastructure.
Usage of Github actions or other CI technologies must be studied in order to provide this feature with proper support for users.

## Reason and benefit

- There is a need from our user base for this as expressed in #371 
- Generally speaking, the Eradiate team wants to reach a larger user base, and we should not rely on WSL to reach MS windows users.
- It would facilitate Eradiate installation for some people

## Details

- The Mitsuba kernel and its wheel must be compiled and uploaded to PyPI for Windows
- We need to define a way to test Eradiate under Windows
- We potentially need dedicated hardware in the dev team

**Comment by wint3ria** [comment]
Windows binaries are currently being compiled on a release candidate for the eradiate-mitsuba kernel. The plan is to ship support for Windows with the upcoming 0.25.0 release of Eradiate

**Comment by wint3ria** [comment]
There is a release candidate available version `0.25.0rc9`, featuring Windows support for Python `3.8`, `3.9`, `3.10` and `3.11`.

I faced some compilation issues with Python `3.12` that I could not figure out at the moment, but it is not related to the host operating system.

Another issue that I noticed is the variable `ERADIATE_DOWNLOAD_DIR` that is set to an incorrect value. Users can workaround this issue by overwriting it their environment prior to executing eradiate. I would like to fix this before releasing the 0.25.0

Still, this should give some support for people on Windows.

**Comment by wint3ria** [comment]
The version of Drjit we currently use is not provided for Python 3.12, we need to merge the latest updates from the Mitsuba team in order to implement the suport for this python version.

On the other hand, we faced some issues with the compiled version for Windows and want to investigate further before releasing it. 

Thus Windows support will not be part of the Eradiate 0.25.0

**Comment by wint3ria** [comment]
The release candidate v0.2.0rc2 features windows binaries for python 3.8, 3.9, 3.10, 3.11 and 3.12. It will be used on the next branch after #386 is merged. I need to find some time in order to test the Windows binaries

**Comment by wint3ria** [comment]
Next steps for the Windows support are the upgrade of the mitsuba submodule, and a manual testing phase on several OS-python version combinations
